apps/web/utils/ai/rule/create-rule-schema.ts

```diff
@@ -76,6 +77,14 @@ const actionSchema = z.object({
     .describe(
       "The fields to use for the action. Static text can be combined with dynamic values using double braces {{}}. For example: 'Hi {{sender's name}}' or 'Re: {{subject}}' or '{{when I'm available for a meeting}}'. Dynamic values will be replaced with actual email data when the rule is executed. Dynamic values are generated in real time by the AI. Only use dynamic values where absolutely necessary. Otherwise, use plain static text. A field can be also be fully static or fully dynamic.",
     ),
+  delayInMinutes: z
+    .number()
+    .min(1, "Minimum supported delay is 1 minute")
+    .max(NINETY_DAYS_MINUTES, "Maximum supported delay is 90 days")
+    .nullish()
+    .describe(
+      "Optional delay in minutes (1 min to 90 days) before executing this action",
+    ),
```

Comment:
3rd time we have this code. and we don't need the validation messages here anyway